garak/generators/openai.py

```diff
@@ -135,6 +135,8 @@ def __init__(self, name="", generations=10, config_root=_config):
         self._load_config(config_root)
         self.fullname = f"{self.generator_family_name} {self.name}"
         self.key_env_var = self.ENV_VAR
+        if "seed" not in dir(self):
```

Comment:
Prefer `hasattr` vs `dir` see [here](https://stackoverflow.com/questions/17723569/whats-the-difference-between-hasattr-and-attribute-in-dir) for reasoning: 
```suggestion
        if not hasattr(self, "seed"):
```